src/pages/Bridge/IssueForm/index.tsx

```diff
@@ -178,44 +184,9 @@ const IssueForm = (): JSX.Element | null => {
   }
 
   if (status === STATUSES.RESOLVED) {
-    const fetchVaults = async (): Promise<Map<AccountId, BitcoinAmount> | undefined> => {
-      try {
-        // This data (the vaults) is strictly required to request issue
-        const vaults = await window.bridge.vaults.getVaultsWithIssuableTokens();
-
-        return vaults;
-      } catch (error) {
-        // We need to reset here as this function is called
-        // on form submit
-        if (submitStatus === STATUSES.PENDING) {
-          setSubmitStatus(STATUSES.RESOLVED);
-        }
-
-        setStatus(STATUSES.REJECTED);
-        handleError(error);
-      }
-    };
-
-    const validateVaultCapacity = async (btcAmount: BitcoinAmount): Promise<string | undefined> => {
-      const vaults = await fetchVaults();
-      const vaultId = getRandomVaultIdWithCapacity(Array.from(vaults || new Map()), btcAmount);
-
-      let vaultMaxAmount = BitcoinAmount.zero;
-      // The first item is the vault with the largest capacity
-      vaultMaxAmount = vaults?.values().next().value;
```

Comment:
It looks like this behavior was changed in the lib: https://github.com/interlay/interbtc-api/blob/master/src/parachain/vaults.ts#L149

Instead of in line 184 `const vaults = await window.bridge.vaults.getVaultsWithIssuableTokens();`, we should use `window.bridge.issue.getRequestLimits` from 
https://github.com/interlay/interbtc-api/blob/master/src/parachain/issue.ts#L52 to get the issuable capacity or we could modify the util function `getRandomVaultIdWithCapacity` to also return the max issuable capacity.

Reply:
Thank you for letting me know that API which has made the process simpler.